packages/checkout-storefront/src/sections/Summary/SummaryItemMoneyEditableSection.tsx

```diff
@@ -1,29 +1,35 @@
 import clsx from "clsx";
-import { debounce } from "lodash-es";
-import React, { useCallback, useEffect, useRef, useState } from "react";
-
-import { IconButton } from "@/checkout-storefront/components/IconButton";
-import { MinusIcon, PlusIcon } from "@/checkout-storefront/icons";
 import { Text } from "@saleor/ui-kit";
 import {
   CheckoutLineFragment,
   CheckoutLinesUpdateMutationVariables,
+  useCheckoutLineDeleteMutation,
   useCheckoutLinesUpdateMutation,
 } from "@/checkout-storefront/graphql";
 import { useFormattedMessages } from "@/checkout-storefront/hooks/useFormattedMessages";
-import { useFormattedMoney } from "@/checkout-storefront/hooks/useFormattedMoney";
 import { Money } from "@/checkout-storefront/components/Money";
+import { TextInput } from "@/checkout-storefront/components/TextInput";
+
+import { useEffect } from "react";
+import {
+  extractMutationErrors,
+  getFormattedMoney,
+  useValidationResolver,
+} from "@/checkout-storefront/lib/utils";
 import { useCheckout } from "@/checkout-storefront/hooks/useCheckout";
 import { useAlerts } from "@/checkout-storefront/hooks/useAlerts";
-import { extractMutationErrors } from "@/checkout-storefront/lib/utils";
-import { getSvgSrc } from "@/checkout-storefront/lib/svgSrc";
+import { object, string } from "yup";
+import { useForm } from "react-hook-form";
+import { useGetInputProps } from "@/checkout-storefront/hooks/useGetInputProps";
+import { useErrors } from "@/checkout-storefront/hooks/useErrors";
+import { Skeleton } from "@/checkout-storefront/components";
 
 interface LineItemQuantitySelectorProps {
   line: CheckoutLineFragment;
 }
 
-interface FormData {
-  quantity: number;
+export interface FormData {
```

Comment:
Let's move `schema` here and infer `FormData` from it.

Reply:
Actually noticed quantity should be required so left it as it is since we need to translate a message and provide it to a schema in the component